packages/front-end/components/Archetype/ArchetypeList.tsx

```diff
@@ -52,7 +52,7 @@ export const ArchetypeList: FC<{
             <h2>Create Reusable Archetypes</h2>
             <p>
               Archetypes are named sets of attributes that help you test your
-              features. Archetypes are a premium feature.
+              features. Archetypes are a Pro feature.
```

Comment:
We should not hardcode this since it will become out of date if we change the plan for this feature later.